### PR TITLE
Feat loader

### PR DESCRIPTION
**Pull request by guillaumewttj**
This loader look like the loader into this button : https://codepen.io/eriscolaro/pen/RwWbMVM

### CODE REVIEW

File: packages/Loader/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <S.LoadingDot backgroundColor={color} shape="circle" size={size} />
+    <S.LoadingDot backgroundColor={color} shape="circle" size={size} />
+  </S.LoaderWrapper>
+))
+
+Loader.displayName = 'Loader'
+
+Loader.propTypes = {
+  color: string.isRequired,
+  size: oneOf('sm', 'md', 'lg')
```
Comment (theo-mesnil) [inline packages/Loader/index.js:18]:
```suggestion
  size: oneOf(['sm', 'md', 'lg'])
```

File: packages/Loader/styles.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ({ size }) =>
+    css`
+      width: ${th(`loaders.sizes.${size}`)};
+      height: ${th(`loaders.sizes.${size}`)};
+    `
+)
+
+export const LoaderWrapper = styled.div`
+  display: flex;
+  > ${LoadingDot} {
```
Comment (theo-mesnil) [inline packages/Loader/styles.js:26]: line break, c'est obligé le `>` ?

File: packages/Loader/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react'
+import { oneOf, string } from 'prop-types'
+
+import * as S from './styles'
+
+export const Loader = forwardRef(({ color, dataTestId, size = 'sm', ...rest }, ref) => (
+  <S.LoaderWrapper data-testid={dataTestId} ref={ref} {...rest}>
+    <S.LoadingDot backgroundColor={color} shape="circle" size={size} />
```
Comment (theo-mesnil) [inline packages/Loader/index.js:8]: add a color by default on theme, and add the possibilty to pass a width and height props if we want to put another size custom ?

File: packages/Loader/styles.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+export const LoadingDot = styled(Shape)(
+  ({ size }) =>
+    css`
+      width: ${th(`loaders.sizes.${size}`)};
+      height: ${th(`loaders.sizes.${size}`)};
+    `
+)
+
+export const LoaderWrapper = styled.div`
+  display: flex;
```
Comment (theo-mesnil) [inline packages/Loader/styles.js:25]:
```suggestion
  display: flex;
  ${system};
```

File: packages/Loader/doc.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+<TagVersion version={version} name={name} />
+
+## Install and import
+
+<Usage component={component} name={name} peerDependencies={peerDependencies} />
+
+## Examples
+
+<Playground>
+  <Loader color="black" />
```
Comment (theo-mesnil) [inline packages/Loader/doc.mdx:23]: remove color here, and add an exemple with color property

File: packages/Loader/styles.js (hunk 2 above)
Comment (Calvein) [reply packages/Loader/styles.js:26]: Je rejoins Théo, à priori il n'y a pas besoin de nester `LoadingDot` dans `LoadingWrapper`

File: packages/Loader/index.js (hunk 3 above)
Comment (guillaumewttj) [reply packages/Loader/index.js:8]: the size is configured with the theme so you can create your own width/height for any size

Comment (theo-mesnil) [reply packages/Loader/index.js:8]: yes i know, but we need to do the same everywhere and for exemple an icon can have a size prop or width and height for special case

**Pull request merged**